assets/sass/_container.scss

```diff
@@ -0,0 +1,33 @@
+.auth-container {
+  border-radius: 3px;
+  border-style: solid;
+  border-width: 1px;
+  height: auto;
+  margin: 100px auto 8px;
+  width: $container-width;
+  min-width: $container-min-width;
+  @include box-sizing(border-box);
+
+  @include device-mq(x-small) {
+    margin-top: 0;
+    width: 100%;
+  }
+
+  @include max-width-mq($container-width) {
+    width: 100%;
+  }
+
+  @include max-height-mq(750px) {
+    margin-top: 0;
+  }
+}
+
+.no-beacon .auth-content {
```

Comment:
Does this live on the same level as auth-container? If so, we should also scope this with #okta-sign-in. If not, let's add it inside the body of the .auth-container class.

Actually in general, I am wondering why we need to set the styles to .auth-container - what happens if we just use #okta-signin-in instead?